website_product_filters/tests/test_has_products_category.py

```diff
@@ -50,3 +49,14 @@ def test_01_has_products_ok(self):
 
         # Tes Category has not products
         self.assertFalse(self.category_graphics.has_products_ok)
+
+
+class TestProduct(TransactionCase):
+    def setUp(self):
+        super(TestProduct, self).setUp()
+        self.product_iphone = self.env.ref(
+            'product.product_product_29_product_template')
+
+    def test_views(self):
+        self.product_iphone.write({'views': 1})
+        self.assertEqual(1, self.product_iphone.views)
```

Comment:
No, sorry I explain myself incomplete.

When you do a test:

```
def test_NUMBER_name(self):
    """One line description "What are you testing and why""""
    # ....
    # prepare stuff
    # ...
    assertXXX(param1, param2, ...paramN,
                      '''Message explaining what fails''')
```

I am talking about the 'Message explaining what fails''